### PR TITLE
Drivetrain fixes

### PR DESCRIPTION
**Pull request by Irishumanoid**
Cherrypicks drivetrain commits to reset gyro and add correct swerve module offsets.

### CODE REVIEW

File: src/main/java/frc/robot/constants/RobotConstants.java [hunk 1]
```diff
@@ -14,10 +14,12 @@ public static final class NeoMotorConstants {
   public static final class DriveConstants {
     // placeholder CAN IDs, fix these later
 
-    public static final double kFrontLeftChassisAngularOffset = 0.0;
-    public static final double kFrontRightChassisAngularOffset = 0.0;
-    public static final double kBackLeftChassisAngularOffset = 0.0;
-    public static final double kBackRightChassisAngularOffset = 0.0;
+    public static final double kMoveInputRotationalOffset = -Math.PI / 2;
```
Comment (EmDash00) [inline src/main/java/frc/robot/constants/RobotConstants.java]: Remove this through a cherrypick of a commit from the revived drivetrain.

**Review by EmDash00 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/RobotConstants.java (hunk 1 above)
Comment (Irishumanoid) [reply src/main/java/frc/robot/constants/RobotConstants.java:17]: Which commit?

Comment (Irishumanoid) [reply src/main/java/frc/robot/constants/RobotConstants.java:17]: Probably 5c64787

Comment (Irishumanoid) [reply src/main/java/frc/robot/constants/RobotConstants.java]: I can't find a good commit for this. Would it be fine to make changes directly on this branch and resolve minor merge conflicts when we rebase develop into drivetrain or PR drivetrain into develop?

Comment (EmDash00) [reply src/main/java/frc/robot/constants/RobotConstants.java]: No, wait for Joseph to recreate drivetrain. He plans on recreating it by rebranching off of develop and one of the commits in the process will delete this line. If you need to expedite, go expedite Joseph as controls lead.

Comment (Irishumanoid) [reply src/main/java/frc/robot/constants/RobotConstants.java]: Due to lack of time, I'm going to resolve this conversation with appropriate changes done directly on this branch.

File: src/main/java/frc/robot/subsystems/drive/Drivetrain.java [hunk 2]
```diff
@@ -235,6 +242,7 @@ private void move(Vector spdVec, double rot) {
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
     Vector spdCommanded = spdVec;
```
Comment (EmDash00) [inline src/main/java/frc/robot/subsystems/drive/Drivetrain.java]: `spdCommanded` is no longer necessary I believe. Remove.

**Review by EmDash00 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/RobotConstants.java [hunk 3]
```diff
@@ -14,10 +14,10 @@ public static final class NeoMotorConstants {
   public static final class DriveConstants {
     // placeholder CAN IDs, fix these later
```
Comment (garrettwitowski) [inline src/main/java/frc/robot/constants/RobotConstants.java]: Can we remove this comment? Or at least make a trello card to remove this? It's not critical but it'd be nice to remove unnecessary comments.

**Review by EmDash00 — APPROVED** [review]

**Pull request merged**